src/tingle.css

```diff
@@ -134,6 +133,7 @@
 
 .tingle-modal--visible .tingle-modal-box {
   transform: scale(1);
+  transition: all .3s cubic-bezier(.175, .885, .32, 1.275);
```

Comment:
I think it would be better not to add a transition on close (by default).
In my opinion, too much animations degrade UX and I prefer keeping an animation when the window opens to highlight the change of context (modals can be painful so it's a good thing to guide the user).

In any case, it's now possible with your fix to use an animation on close if necessary.